front-end/src/components/common/Loader/Loader.jsx

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
+import { useLocation } from 'react-router-dom';
 
 //Styles
 import styles from './Loader.module.scss';
 
 const Loader = () => {
-  return <div className={styles.loader}></div>;
+  const { pathname } = useLocation();
+  return (
+    <div
+      className={`${styles.loader} ${
+        pathname === '/login' || pathname === '/registration' ? styles.white : styles.purple
```

Comment:
If you need to react on some specific data, you could pass this data to props

Reply:
fix it